src/openssl/ssl/method.cr

```diff
@@ -0,0 +1,18 @@
+enum OpenSSL::SSL::Method
```

Comment:
I"m not sure we need an enum. LibSSL::SSLMethod is already a type. Also, there are a lot more methods in OpenSSL, thought I'm sure why there are so many.